src/lib/isMobilePhone.js

```diff
@@ -105,6 +105,7 @@ const phones = {
   'hu-HU': /^(\+?36|06)(20|30|31|50|70)\d{7}$/,
   'id-ID': /^(\+?62|0)8(1[123456789]|2[1238]|3[1238]|5[12356789]|7[78]|9[56789]|8[123456789])([\s?|\d]{5,11})$/,
   'ir-IR': /^(\+98|0)?9\d{9}$/,
+  'is-IS': /^(\+354)(61[1-8]|62[0-6]|63[0,2]|64[4,6,7,9]|65[0,1,5,9]|66[0-6,9]|670|68[0,6-8]|69[0-9]|7[6-8][0-9]|79[0-3]|82[0-5,9]|8[3-6][0-9]|88[2,3,8]|89[0-9])\d{4}$/,
```

Comment:
I don't think there is an "official" style guideline for how to express RegExp classes in this project, but it looks like the majority of RegExps here are not using commas in their ranges, e.g.:

* `65[0,1,5,9]` -> `65[0159]`
* `66[0-6,9]` -> `66[0-69]` (although I do have to admit, that separating the single digit with a comma here, does kinda look nice and might be easier to understand)

my feeling would be that it is good to try to stick to the style of the other regexp as well- what do you think?

Reply:
oh, right, my brain also must've still been sleeping, when I checked this, thinking that the comma is a valid "separator" in a regexp range :-D
Thanks for fixing it!